### PR TITLE
Fix: Google Analytics module (0.71.7)

### PR DESCRIPTION
**Pull request by cbshoaib**
# Update-Readme: Google Analytics module

## Type of PR

- [ ] Bugfix
- [ ] New feature
- [x] Minor changes

## Changes introduced
add module scope features, dependencies installation, module specifications, contributing and license in README.md file.

### CODE REVIEW

**Review by danielsousaio — COMMENTED** [review]
Needs updated README.md

**Review by saad-abid-crowdbotics — DISMISSED** [review]
LGTM

File: modules/react-native-google-analytics/README.md [hunk 1]
```diff
@@ -45,440 +91,86 @@ dependencies {
...
+On the Firebase console, add a new iOS application and enter your projects details. The "iOS bundle ID" must match your local project bundle ID. The bundle ID can be found within the "General" tab when opening the project with Xcode.
+Download the `GoogleService-Info.plist` file.
+
+1. Using Xcode, open the projects /ios/{projectName}.xcodeproj file (or /ios/{projectName}.xcworkspace if using Pods).
+
+   Right click on the project name and "Add files" to the project, as demonstrated below:
+   ![image](https://github.com/cbshoaib/modules/assets/76822297/d9e5db27-4279-4c96-bbe8-c27b79691e68)
+
+Select the downloaded GoogleService-Info.plist file from your computer, and ensure the "Copy items if needed" checkbox is enabled.
+![image](https://github.com/cbshoaib/modules/assets/76822297/8f9c8db2-9d3a-4b51-825d-8f36bc112099)
```
Comment (danielsousaio) [inline modules/react-native-google-analytics/README.md:105]: Could you please upload those to this repo?

**Review by danielsousaio — CHANGES_REQUESTED** [review]

**Review by danielsousaio — APPROVED** [review]

**Review by danielsousaio — CHANGES_REQUESTED** [review]
Missing images upload

File: modules/react-native-google-analytics/README.md (hunk 1 above)
Comment (saad-abid-crowdbotics) [reply modules/react-native-google-analytics/README.md]: These images are also present in the official documentation of react-native-firebase. I have replaced the image links with the ones that are being used in that documentation.

[documentation](https://rnfirebase.io/)

**Review by danielsousaio — APPROVED** [review]

**Pull request merged**